docs/_templates/sidebarintro.html

```diff
@@ -46,20 +46,6 @@ <h3>Other Projects</h3>
   <li><a href="https://pipenv-pipes.readthedocs.io/en/latest/">Pipenv-Pipes</a></li>
 </ul>
 
-<p>More projects founded by <a href="https://kenreitz.org">Kenneth Reitz</a>:</p>
-<ul>
-    <li><a href="https://pep8.org/">pep8.org</a></li>
-    <li><a href="https://httpbin.org/">httpbin.org</a></li>
-    <li><a href="https://howtopython.org">HowToPython.org</a></li>
-    <li><a href="https://python-guide.org">The Python Guide</a></li>
-    <li><a href="https://requests.readthedocs.io">Requests: HTTP for Humans</a></li>
```

Comment:
requests is still alive. I am fine though with removing the "advertisement for KR" ...